crawl-ref/source/cloud.cc

```diff
@@ -918,6 +918,12 @@ bool actor_cloud_immune(const actor &act, const cloud_struct &cloud)
         return true;
     }
 
+    if (!player && have_passive(passive_t::cloud_immunity)
+        && (act.as_monster()->friendly() || act.as_monster()->neutral()))
+    {
+        return true;
+    }
+
```

Comment:
These braces are unnecessary when you have a single line inside the `if`.

Reply:
I'll forward this by pointing out I'm still relatively new to programming and my C++ experience starts and ends with the few months I've learned via modding Crawl. In this case I borrowed the formatting from the block just below this one, and although I recognized it as unnecessary, I thought it helped with readability following a multi-line if statement. Is this a reasonable approach, or is a hard rule of no bracing a standard practice I should be following regardless of circumstance?